`bevy_ui` does not compile without `bevy_text` feature

## Bevy version

main (`ee9a1503edb6ff72cc69514c6336d9f624f0d600`)

## What you did
Run `cargo build -p bevy_ui`

## What went wrong

```
   Compiling bevy_ui v0.12.0 (/home/chrisjuchem/bevy/crates/bevy_ui)
error[E0425]: cannot find value `text_system` in module `widget`
   --> crates/bevy_ui/src/lib.rs:200:45
    |
200 |                     .ambiguous_with(widget::text_system),
    |                                             ^^^^^^^^^^^ not found in `widget`

error[E0425]: cannot find value `text_system` in module `widget`
   --> crates/bevy_ui/src/lib.rs:207:45
    |
207 |                     .ambiguous_with(widget::text_system),
    |                                             ^^^^^^^^^^^ not found in `widget`

For more information about this error, try `rustc --explain E0425`.
error: could not compile `bevy_ui` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```

## Additional information

Looks like this system only exists behind the `bevy_text` feature, but the ambiguity declaration was not put behind the same feature in #10411.
Building succeeds with `cargo build -p bevy_ui --features bevy_text`.

--- a/crates/bevy_ui/src/lib.rs
+++ b/crates/bevy_ui/src/lib.rs
@@ -186,8 +186,8 @@ impl Plugin for UiPlugin {
 /// A function that should be called from [`UiPlugin::build`] when [`bevy_text`] is enabled.
 #[cfg(feature = "bevy_text")]
 fn build_text_interop(app: &mut App) {
-    use bevy_text::TextLayoutInfo;
     use crate::widget::TextFlags;
+    use bevy_text::TextLayoutInfo;
 
     app.register_type::<TextLayoutInfo>()
         .register_type::<TextFlags>();

--- a/crates/bevy_ui/src/lib.rs
+++ b/crates/bevy_ui/src/lib.rs
@@ -192,7 +192,6 @@ fn build_text_interop(app: &mut App) {
     app.register_type::<TextLayoutInfo>()
         .register_type::<TextFlags>();
 
-    // add these systems to front because these must run before transform update systems
     app.add_systems(
         PostUpdate,
         (

--- a/crates/bevy_ui/src/lib.rs
+++ b/crates/bevy_ui/src/lib.rs
@@ -132,6 +132,7 @@ impl Plugin for UiPlugin {
         #[cfg(feature = "bevy_text")]
         app.register_type::<TextLayoutInfo>()
             .register_type::<TextFlags>();
+
         // add these systems to front because these must run before transform update systems
         #[cfg(feature = "bevy_text")]
         app.add_systems(
@@ -158,28 +159,17 @@ impl Plugin for UiPlugin {
                     .ambiguous_with(bevy_text::update_text2d_layout),
             ),
         );
+
         #[cfg(feature = "bevy_text")]
         app.add_plugins(accessibility::AccessibilityPlugin);
-        app.add_systems(PostUpdate, {
-            #[allow(clippy::let_and_return)]
-            let system = widget::update_image_content_size_system.before(UiSystem::Layout);
-
-            // Potential conflicts: `Assets<Image>`
-            // They run independently since `widget::image_node_system` will only ever observe
-            // its own UiImage, and `widget::text_system` & `bevy_text::update_text2d_layout`
-            // will never modify a pre-existing `Image` asset.
-            #[cfg(feature = "bevy_text")]
-            let system = system
-                .ambiguous_with(bevy_text::update_text2d_layout)
-                .ambiguous_with(widget::text_system);
-
-            system
-        });
 
         // Marks systems that can be ambiguous with [`widget::text_system`] if the `bevy_text` feature is enabled.
         // See https://github.com/bevyengine/bevy/pull/11391 for more details.
         #[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
         struct AmbiguousWithTextSystem;
+
+        #[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
+        struct AmbiguousWithUpdateText2DLayout;
 
         app.add_systems(
             PostUpdate,
@@ -205,6 +195,14 @@ impl Plugin for UiPlugin {
                     .ambiguous_with(ui_layout_system)
                     .in_set(AmbiguousWithTextSystem),
                 update_clipping_system.after(TransformSystem::TransformPropagate),
+                // Potential conflicts: `Assets<Image>`
+                // They run independently since `widget::image_node_system` will only ever observe
+                // its own UiImage, and `widget::text_system` & `bevy_text::update_text2d_layout`
+                // will never modify a pre-existing `Image` asset.
+                widget::update_image_content_size_system
+                    .before(UiSystem::Layout)
+                    .in_set(AmbiguousWithTextSystem)
+                    .in_set(AmbiguousWithUpdateText2DLayout),
             ),
         );
 
@@ -212,6 +210,12 @@ impl Plugin for UiPlugin {
         app.configure_sets(
             PostUpdate,
             AmbiguousWithTextSystem.ambiguous_with(widget::text_system),
+        );
+
+        #[cfg(feature = "bevy_text")]
+        app.configure_sets(
+            PostUpdate,
+            AmbiguousWithUpdateText2DLayout.ambiguous_with(bevy_text::update_text2d_layout),
         );
 
         build_ui_render(app);

--- a/crates/bevy_ui/src/lib.rs
+++ b/crates/bevy_ui/src/lib.rs
@@ -161,7 +161,9 @@ impl Plugin for UiPlugin {
         #[cfg(feature = "bevy_text")]
         app.add_plugins(accessibility::AccessibilityPlugin);
         app.add_systems(PostUpdate, {
+            #[allow(clippy::let_and_return)]
             let system = widget::update_image_content_size_system.before(UiSystem::Layout);
+
             // Potential conflicts: `Assets<Image>`
             // They run independently since `widget::image_node_system` will only ever observe
             // its own UiImage, and `widget::text_system` & `bevy_text::update_text2d_layout`
@@ -184,19 +186,33 @@ impl Plugin for UiPlugin {
                 ui_layout_system
                     .in_set(UiSystem::Layout)
                     .before(TransformSystem::TransformPropagate),
-                resolve_outlines_system
-                    .in_set(UiSystem::Outlines)
-                    .after(UiSystem::Layout)
-                    // clipping doesn't care about outlines
-                    .ambiguous_with(update_clipping_system)
-                    .ambiguous_with(widget::text_system),
-                ui_stack_system
-                    .in_set(UiSystem::Stack)
-                    // the systems don't care about stack index
-                    .ambiguous_with(update_clipping_system)
-                    .ambiguous_with(resolve_outlines_system)
-                    .ambiguous_with(ui_layout_system)
-                    .ambiguous_with(widget::text_system),
+                {
+                    #[allow(clippy::let_and_return)]
+                    let system = resolve_outlines_system
+                        .in_set(UiSystem::Outlines)
+                        .after(UiSystem::Layout)
+                        // clipping doesn't care about outlines
+                        .ambiguous_with(update_clipping_system);
+
+                    #[cfg(feature = "bevy_text")]
+                    let system = system.ambiguous_with(widget::text_system);
+
+                    system
+                },
+                {
+                    #[allow(clippy::let_and_return)]
+                    let system = ui_stack_system
+                        .in_set(UiSystem::Stack)
+                        // the systems don't care about stack index
+                        .ambiguous_with(update_clipping_system)
+                        .ambiguous_with(resolve_outlines_system)
+                        .ambiguous_with(ui_layout_system);
+
+                    #[cfg(feature = "bevy_text")]
+                    let system = system.ambiguous_with(widget::text_system);
+
+                    system
+                },
                 update_clipping_system.after(TransformSystem::TransformPropagate),
             ),
         );

--- a/crates/bevy_ui/src/lib.rs
+++ b/crates/bevy_ui/src/lib.rs
@@ -176,6 +176,11 @@ impl Plugin for UiPlugin {
             system
         });
 
+        /// Marks systems that can be ambiguous with [`widget::text_system`], if the `bevy_text` feature is enabled.
+        /// https://github.com/bevyengine/bevy/pull/11391
+        #[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
+        struct AmbiguousWithTextSystem;
+
         app.add_systems(
             PostUpdate,
             (
@@ -186,35 +191,27 @@ impl Plugin for UiPlugin {
                 ui_layout_system
                     .in_set(UiSystem::Layout)
                     .before(TransformSystem::TransformPropagate),
-                {
-                    #[allow(clippy::let_and_return)]
-                    let system = resolve_outlines_system
-                        .in_set(UiSystem::Outlines)
-                        .after(UiSystem::Layout)
-                        // clipping doesn't care about outlines
-                        .ambiguous_with(update_clipping_system);
-
-                    #[cfg(feature = "bevy_text")]
-                    let system = system.ambiguous_with(widget::text_system);
-
-                    system
-                },
-                {
-                    #[allow(clippy::let_and_return)]
-                    let system = ui_stack_system
-                        .in_set(UiSystem::Stack)
-                        // the systems don't care about stack index
-                        .ambiguous_with(update_clipping_system)
-                        .ambiguous_with(resolve_outlines_system)
-                        .ambiguous_with(ui_layout_system);
-
-                    #[cfg(feature = "bevy_text")]
-                    let system = system.ambiguous_with(widget::text_system);
-
-                    system
-                },
+                resolve_outlines_system
+                    .in_set(UiSystem::Outlines)
+                    .after(UiSystem::Layout)
+                    // clipping doesn't care about outlines
+                    .ambiguous_with(update_clipping_system)
+                    .in_set(AmbiguousWithTextSystem),
+                ui_stack_system
+                    .in_set(UiSystem::Stack)
+                    // the systems don't care about stack index
+                    .ambiguous_with(update_clipping_system)
+                    .ambiguous_with(resolve_outlines_system)
+                    .ambiguous_with(ui_layout_system)
+                    .in_set(AmbiguousWithTextSystem),
                 update_clipping_system.after(TransformSystem::TransformPropagate),
             ),
+        );
+
+        #[cfg(feature = "bevy_text")]
+        app.configure_sets(
+            PostUpdate,
+            AmbiguousWithTextSystem.ambiguous_with(widget::text_system),
         );
 
         build_ui_render(app);

--- a/crates/bevy_ui/src/lib.rs
+++ b/crates/bevy_ui/src/lib.rs
@@ -176,8 +176,8 @@ impl Plugin for UiPlugin {
             system
         });
 
-        /// Marks systems that can be ambiguous with [`widget::text_system`], if the `bevy_text` feature is enabled.
-        /// https://github.com/bevyengine/bevy/pull/11391
+        // Marks systems that can be ambiguous with [`widget::text_system`] if the `bevy_text` feature is enabled.
+        // See https://github.com/bevyengine/bevy/pull/11391 for more details.
         #[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
         struct AmbiguousWithTextSystem;
 

--- a/crates/bevy_ui/src/lib.rs
+++ b/crates/bevy_ui/src/lib.rs
@@ -12,8 +12,6 @@ pub mod widget;
 use bevy_derive::{Deref, DerefMut};
 use bevy_reflect::Reflect;
 #[cfg(feature = "bevy_text")]
-use bevy_text::TextLayoutInfo;
-#[cfg(feature = "bevy_text")]
 mod accessibility;
 mod focus;
 mod geometry;
@@ -40,8 +38,6 @@ pub mod prelude {
     };
 }
 
-#[cfg(feature = "bevy_text")]
-use crate::widget::TextFlags;
 use bevy_app::prelude::*;
 use bevy_ecs::prelude::*;
 use bevy_input::InputSystem;
@@ -80,6 +76,14 @@ impl Default for UiScale {
         Self(1.0)
     }
 }
+
+// Marks systems that can be ambiguous with [`widget::text_system`] if the `bevy_text` feature is enabled.
+// See https://github.com/bevyengine/bevy/pull/11391 for more details.
+#[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
+struct AmbiguousWithTextSystem;
+
+#[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
+struct AmbiguousWithUpdateText2DLayout;
 
 impl Plugin for UiPlugin {
     fn build(&self, app: &mut App) {
@@ -129,48 +133,6 @@ impl Plugin for UiPlugin {
                 ui_focus_system.in_set(UiSystem::Focus).after(InputSystem),
             );
 
-        #[cfg(feature = "bevy_text")]
-        app.register_type::<TextLayoutInfo>()
-            .register_type::<TextFlags>();
-
-        // add these systems to front because these must run before transform update systems
-        #[cfg(feature = "bevy_text")]
-        app.add_systems(
-            PostUpdate,
-            (
-                widget::measure_text_system
-                    .before(UiSystem::Layout)
-                    // Potential conflict: `Assets<Image>`
-                    // In practice, they run independently since `bevy_render::camera_update_system`
-                    // will only ever observe its own render target, and `widget::measure_text_system`
-                    // will never modify a pre-existing `Image` asset.
-                    .ambiguous_with(bevy_render::camera::CameraUpdateSystem)
-                    // Potential conflict: `Assets<Image>`
-                    // Since both systems will only ever insert new [`Image`] assets,
-                    // they will never observe each other's effects.
-                    .ambiguous_with(bevy_text::update_text2d_layout)
-                    // We assume Text is on disjoint UI entities to UiImage and UiTextureAtlasImage
-                    // FIXME: Add an archetype invariant for this https://github.com/bevyengine/bevy/issues/1481.
-                    .ambiguous_with(widget::update_image_content_size_system),
-                widget::text_system
-                    .after(UiSystem::Layout)
-                    .after(bevy_text::remove_dropped_font_atlas_sets)
-                    // Text2d and bevy_ui text are entirely on separate entities
-                    .ambiguous_with(bevy_text::update_text2d_layout),
-            ),
-        );
-
-        #[cfg(feature = "bevy_text")]
-        app.add_plugins(accessibility::AccessibilityPlugin);
-
-        // Marks systems that can be ambiguous with [`widget::text_system`] if the `bevy_text` feature is enabled.
-        // See https://github.com/bevyengine/bevy/pull/11391 for more details.
-        #[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
-        struct AmbiguousWithTextSystem;
-
-        #[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
-        struct AmbiguousWithUpdateText2DLayout;
-
         app.add_systems(
             PostUpdate,
             (
@@ -207,16 +169,7 @@ impl Plugin for UiPlugin {
         );
 
         #[cfg(feature = "bevy_text")]
-        app.configure_sets(
-            PostUpdate,
-            AmbiguousWithTextSystem.ambiguous_with(widget::text_system),
-        );
-
-        #[cfg(feature = "bevy_text")]
-        app.configure_sets(
-            PostUpdate,
-            AmbiguousWithUpdateText2DLayout.ambiguous_with(bevy_text::update_text2d_layout),
-        );
+        build_text_interop(app);
 
         build_ui_render(app);
     }
@@ -228,4 +181,52 @@ impl Plugin for UiPlugin {
 
         render_app.init_resource::<UiPipeline>();
     }
+}
+
+/// A function that should be called from [`UiPlugin::build`] when [`bevy_text`] is enabled.
+#[cfg(feature = "bevy_text")]
+fn build_text_interop(app: &mut App) {
+    use bevy_text::TextLayoutInfo;
+    use crate::widget::TextFlags;
+
+    app.register_type::<TextLayoutInfo>()
+        .register_type::<TextFlags>();
+
+    // add these systems to front because these must run before transform update systems
+    app.add_systems(
+        PostUpdate,
+        (
+            widget::measure_text_system
+                .before(UiSystem::Layout)
+                // Potential conflict: `Assets<Image>`
+                // In practice, they run independently since `bevy_render::camera_update_system`
+                // will only ever observe its own render target, and `widget::measure_text_system`
+                // will never modify a pre-existing `Image` asset.
+                .ambiguous_with(bevy_render::camera::CameraUpdateSystem)
+                // Potential conflict: `Assets<Image>`
+                // Since both systems will only ever insert new [`Image`] assets,
+                // they will never observe each other's effects.
+                .ambiguous_with(bevy_text::update_text2d_layout)
+                // We assume Text is on disjoint UI entities to UiImage and UiTextureAtlasImage
+                // FIXME: Add an archetype invariant for this https://github.com/bevyengine/bevy/issues/1481.
+                .ambiguous_with(widget::update_image_content_size_system),
+            widget::text_system
+                .after(UiSystem::Layout)
+                .after(bevy_text::remove_dropped_font_atlas_sets)
+                // Text2d and bevy_ui text are entirely on separate entities
+                .ambiguous_with(bevy_text::update_text2d_layout),
+        ),
+    );
+
+    app.add_plugins(accessibility::AccessibilityPlugin);
+
+    app.configure_sets(
+        PostUpdate,
+        AmbiguousWithTextSystem.ambiguous_with(widget::text_system),
+    );
+
+    app.configure_sets(
+        PostUpdate,
+        AmbiguousWithUpdateText2DLayout.ambiguous_with(bevy_text::update_text2d_layout),
+    );
 }

--- a/crates/bevy_ui/src/render/mod.rs
+++ b/crates/bevy_ui/src/render/mod.rs
@@ -8,24 +8,18 @@ use bevy_render::{
     render_phase::PhaseItem, render_resource::BindGroupEntries, view::ViewVisibility,
     ExtractSchedule, Render,
 };
-use bevy_sprite::SpriteAssetEvents;
-#[cfg(feature = "bevy_text")]
-use bevy_sprite::TextureAtlas;
+use bevy_sprite::{SpriteAssetEvents, TextureAtlas};
 pub use pipeline::*;
 pub use render_pass::*;
 pub use ui_material_pipeline::*;
 
-#[cfg(feature = "bevy_text")]
-use crate::{BackgroundColor, UiImage};
 use crate::{
-    BorderColor, CalculatedClip, ContentSize, DefaultUiCamera, Node, Outline, Style, TargetCamera,
-    UiScale, Val,
+    BackgroundColor, BorderColor, CalculatedClip, ContentSize, DefaultUiCamera, Node, Outline,
+    Style, TargetCamera, UiImage, UiScale, Val,
 };
 
 use bevy_app::prelude::*;
-#[cfg(feature = "bevy_text")]
-use bevy_asset::Assets;
-use bevy_asset::{load_internal_asset, AssetEvent, AssetId, Handle};
+use bevy_asset::{load_internal_asset, AssetEvent, AssetId, Assets, Handle};
 use bevy_ecs::prelude::*;
 use bevy_math::{Mat4, Rect, URect, UVec4, Vec2, Vec3, Vec4Swizzles};
 use bevy_render::{
@@ -40,7 +34,6 @@ use bevy_render::{
     view::{ExtractedView, ViewUniforms},
     Extract, RenderApp, RenderSet,
 };
-#[cfg(feature = "bevy_text")]
 use bevy_sprite::TextureAtlasLayout;
 #[cfg(feature = "bevy_text")]
 use bevy_text::{PositionedGlyph, Text, TextLayoutInfo};
@@ -404,7 +397,6 @@ pub fn extract_uinode_outlines(
     }
 }
 
-#[cfg(feature = "bevy_text")]
 pub fn extract_uinodes(
     mut extracted_uinodes: ResMut<ExtractedUiNodes>,
     images: Extract<Res<Assets<Image>>>,

--- a/crates/bevy_ui/src/render/mod.rs
+++ b/crates/bevy_ui/src/render/mod.rs
@@ -15,16 +15,17 @@ pub use pipeline::*;
 pub use render_pass::*;
 pub use ui_material_pipeline::*;
 
-use crate::{
-    BorderColor, CalculatedClip, ContentSize, Node, Style, UiScale, Val, DefaultUiCamera, Outline, TargetCamera,
-};
 #[cfg(feature = "bevy_text")]
 use crate::{BackgroundColor, UiImage};
+use crate::{
+    BorderColor, CalculatedClip, ContentSize, DefaultUiCamera, Node, Outline, Style, TargetCamera,
+    UiScale, Val,
+};
 
 use bevy_app::prelude::*;
-use bevy_asset::{load_internal_asset, AssetEvent, AssetId, Handle};
 #[cfg(feature = "bevy_text")]
 use bevy_asset::Assets;
+use bevy_asset::{load_internal_asset, AssetEvent, AssetId, Handle};
 use bevy_ecs::prelude::*;
 use bevy_math::{Mat4, Rect, URect, UVec4, Vec2, Vec3, Vec4Swizzles};
 use bevy_render::{

--- a/crates/bevy_ui/src/render/mod.rs
+++ b/crates/bevy_ui/src/render/mod.rs
@@ -81,7 +81,6 @@ pub fn build_ui_render(app: &mut App) {
             (
                 extract_default_ui_camera_view::<Camera2d>,
                 extract_default_ui_camera_view::<Camera3d>,
-                #[cfg(feature = "bevy_text")]
                 extract_uinodes.in_set(RenderUiSystem::ExtractNode),
                 extract_uinode_borders.after(RenderUiSystem::ExtractAtlasNode),
                 #[cfg(feature = "bevy_text")]

--- a/crates/bevy_ui/src/render/mod.rs
+++ b/crates/bevy_ui/src/render/mod.rs
@@ -8,18 +8,23 @@ use bevy_render::{
     render_phase::PhaseItem, render_resource::BindGroupEntries, view::ViewVisibility,
     ExtractSchedule, Render,
 };
-use bevy_sprite::{SpriteAssetEvents, TextureAtlas};
+use bevy_sprite::SpriteAssetEvents;
+#[cfg(feature = "bevy_text")]
+use bevy_sprite::TextureAtlas;
 pub use pipeline::*;
 pub use render_pass::*;
 pub use ui_material_pipeline::*;
 
 use crate::{
-    BackgroundColor, BorderColor, CalculatedClip, ContentSize, Node, Style, UiImage, UiScale, Val,
+    BorderColor, CalculatedClip, ContentSize, Node, Style, UiScale, Val, DefaultUiCamera, Outline, TargetCamera,
 };
-use crate::{DefaultUiCamera, Outline, TargetCamera};
+#[cfg(feature = "bevy_text")]
+use crate::{BackgroundColor, UiImage};
 
 use bevy_app::prelude::*;
-use bevy_asset::{load_internal_asset, AssetEvent, AssetId, Assets, Handle};
+use bevy_asset::{load_internal_asset, AssetEvent, AssetId, Handle};
+#[cfg(feature = "bevy_text")]
+use bevy_asset::Assets;
 use bevy_ecs::prelude::*;
 use bevy_math::{Mat4, Rect, URect, UVec4, Vec2, Vec3, Vec4Swizzles};
 use bevy_render::{
@@ -82,6 +87,7 @@ pub fn build_ui_render(app: &mut App) {
             (
                 extract_default_ui_camera_view::<Camera2d>,
                 extract_default_ui_camera_view::<Camera3d>,
+                #[cfg(feature = "bevy_text")]
                 extract_uinodes.in_set(RenderUiSystem::ExtractNode),
                 extract_uinode_borders.after(RenderUiSystem::ExtractAtlasNode),
                 #[cfg(feature = "bevy_text")]
@@ -397,6 +403,7 @@ pub fn extract_uinode_outlines(
     }
 }
 
+#[cfg(feature = "bevy_text")]
 pub fn extract_uinodes(
     mut extracted_uinodes: ResMut<ExtractedUiNodes>,
     images: Extract<Res<Assets<Image>>>,

--- a/crates/bevy_ui/src/widget/image.rs
+++ b/crates/bevy_ui/src/widget/image.rs
@@ -2,6 +2,7 @@ use crate::{measurement::AvailableSpace, ContentSize, Measure, Node, UiImage, Ui
 use bevy_asset::Assets;
 
 use bevy_ecs::change_detection::DetectChanges;
+#[cfg(feature = "bevy_text")]
 use bevy_ecs::query::Without;
 use bevy_ecs::{
     prelude::Component,

--- a/crates/bevy_ui/src/widget/image.rs
+++ b/crates/bevy_ui/src/widget/image.rs
@@ -1,15 +1,6 @@
 use crate::{measurement::AvailableSpace, ContentSize, Measure, Node, UiImage, UiScale};
 use bevy_asset::Assets;
-
-#[cfg(feature = "bevy_text")]
-use bevy_ecs::query::Without;
-use bevy_ecs::{
-    change_detection::DetectChanges,
-    prelude::Component,
-    query::With,
-    reflect::ReflectComponent,
-    system::{Local, Query, Res},
-};
+use bevy_ecs::prelude::*;
 use bevy_math::Vec2;
 use bevy_reflect::{std_traits::ReflectDefault, Reflect};
 use bevy_render::texture::Image;

--- a/crates/bevy_ui/src/widget/image.rs
+++ b/crates/bevy_ui/src/widget/image.rs
@@ -1,10 +1,10 @@
 use crate::{measurement::AvailableSpace, ContentSize, Measure, Node, UiImage, UiScale};
 use bevy_asset::Assets;
 
-use bevy_ecs::change_detection::DetectChanges;
 #[cfg(feature = "bevy_text")]
 use bevy_ecs::query::Without;
 use bevy_ecs::{
+    change_detection::DetectChanges,
     prelude::Component,
     query::With,
     reflect::ReflectComponent,

